### PR TITLE
upload guide copy changes

### LINKED ISSUE #4404: Update bulk results upload guide (copy changes)
Update guide to incorporate the latest suggestions in this doc: https://docs.google.com/document/d/1EIZvjLOx3PIPA5w1B1oeBKBgqiHRJnMSsLIQHBLysjo/edit?usp=sharing

Changes: 
- [ ] Update link names under resources (p1)
- [ ] Updating data element name to "Ordering provider" (p2) 
- [ ] Updating instructions and linking to device code lookup tool under Equipment model name (p8), Test performed LOINC code(p9), Test result (p10), Specimen type (p12)

### PR DESCRIPTION
**Pull request by DanielSass**
# FRONTEND PULL REQUEST

## Related Issue

- resolves #4404 

## Changes Proposed

- Updating copy, linking to the code lookup page

## Testing

- Look at the guide, compare to what is linked in the ticket

## Screenshots / Demos
Screen Shot 2022-10-31 at 9 58 16 AM
Screen Shot 2022-10-31 at 9 58 42 AM
Screen Shot 2022-10-31 at 9 59 12 AM

### CODE REVIEW

**Comment by DanielSass** [comment]
I think all the lower envs are taken atm, but once one is available I'll push it out and double check everything

**Comment by DanielSass** [comment]
deploying to dev2 now

**Comment by DanielSass** [comment]
https://dev2.simplereport.gov/app/results/upload/submit/guide?facility=78b5bd65-1edd-4a56-ae93-feb0642d1b6f

File: frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx [hunk 1]
```diff
@@ -172,6 +172,25 @@ const CsvSchemaDocumentation = () => {
...
+            relevant state and/or local health department(s).
+            <br />
+            <br />
+            The results uploader is connected with all{" "}
+            <a
+              href="/getting-started/organizations-and-testing-facilities/where-does-simplereport-work"
+              className="usa-link"
+            >
+              states partnered with SimpleReport.
+            </a>{" "}
```
Comment (lauraykerr) [inline frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx:188]:
```suggestion
            The results uploader is connected with all{" "}
            <a
              href="/getting-started/organizations-and-testing-facilities/where-does-simplereport-work"
              className="usa-link"
            >
              states partnered with SimpleReport
            </a>{" "}.
```
Removing period from link to follow content best practices

**Review by lauraykerr — APPROVED** [review]
@DanielSass A couple non-blocking suggestions — one is in a comment, and the other, if possible, is setting the link in the intro ("states partnered with SimpleReport") to open in a new tab since it takes users outside of the logged-in experience.

Otherwise LGTM, thank you!

**Comment by lauraykerr** [comment]
@lg-king Heads up was going back and forth on whether text or video should go first, but I'm fine as is for now unless you have a strong opinion.

**Review by lauraykerr — APPROVED** [review]

**Review by nathancrtr — APPROVED** [review]

**Pull request merged**